app/routes/manager/universities/points-of-contact.tsx

```diff
@@ -0,0 +1,368 @@
+import { useState, useEffect } from "react";
+import { useLoaderData, useNavigation, useRouteError, isRouteErrorResponse } from "@remix-run/react";
+import type { ActionFunction, LoaderFunction } from "@remix-run/node";
+import { json } from "@remix-run/node";
+import Button from "@mui/material/Button";
+import AddIcon from "@mui/icons-material/Add";
+import EditIcon from "@mui/icons-material/Edit";
+import {
+  getPointsOfContact,
+  addPointOfContact,
+  updatePointOfContact,
+} from "~/models/pointsOfContact.server";
+import { Card, CardContent, CardHeader, Container, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, TableSortLabel } from "@mui/material";
+import { ValidatedForm, validationError } from "remix-validated-form";
+import { z } from "zod";
+import { withZod } from "@remix-validated-form/with-zod";
+import ModalBox from "~/core/components/ModalBox";
+import LabeledTextField from "~/core/components/LabeledTextField";
+import { Stack } from "@mui/system";
+import { redirect } from "remix-typedjson";
+import invariant from "tiny-invariant";
+import { validateNavigationRedirect } from '~/utils'
+import { getUniversities } from "~/models/university.server";
+import InputSelect from "~/core/components/InputSelect";
+import ToggleOffIcon from "@mui/icons-material/ToggleOff";
+import ToggleOnIcon from "@mui/icons-material/ToggleOn";
+import { LabeledCheckbox } from "~/core/components/LabeledCheckbox";
+
+declare module "@mui/material/Button" {
+  interface ButtonPropsColorOverrides {
+    secondaryB: true;
+    secondaryC: true;
+  }
+}
+
+export const validator = withZod(
+  z.object({
+    id: z.string().optional(),
+    fullName: z
+      .string()
+      .min(1, { message: "Name is required" }),
+    university: z.object({
+      name: z.string(),
```

Comment:
Adding a .min(1) to the university name on line 43 might work.

Reply:
Added :) Thanks for the suggestion